client/src/pages/GmailDialog.css

```diff
@@ -0,0 +1,7 @@
+#gmail-button {
+  background-image: "/public/google_signin_buttons/web/1x/btn_google_signin_light_normal_web.png"
+}
+
+#gmail-button:focus {
+  background-image: "/public/google_signin_buttons/web/1x/btn_google_signin_light_focus_web.png"
+}
```

Comment:
where is this inputted btw? I also think we should just stick to using makeStyles hook whenever possible